Source/CesiumRuntime/Private/CesiumTileExcluderAdapter.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "Cesium3DTileset.h"
+#include "CesiumTile.h"
+#include "CesiumTileExcluder.h"
+#include <Cesium3DTilesSelection/ITileExcluder.h>
+
+class CesiumTileExcluderAdapter : public Cesium3DTilesSelection::ITileExcluder {
```

Comment:
I think it should be possible to make `CesiumTileExcluder` implement `ITileExcluder`, rather than using an Adapter. The `shouldInclude` method on `ITileExcluder` obviously won't be callable from Blueprints (and in fact it should be private). But I think moving the implementation of the cesium-native interface into CesiumTileExcluder itself will simply things a bit. For example, the raw points in this class make me nervous, because they'll be left dangling if the corresponding UE objects get garbage collected. I'm not certain that can happen, but I'm also not certain that it _can't_. If we only reference UObjects via UPROPERTYs on `CesiumTileExcluder`, we pretty much don't have to worry about it.